radio/src/telemetry/frsky.h

```diff
@@ -36,7 +36,7 @@ enum FrSkyDataState {
 };
 
 #define FRSKY_SPORT_BAUDRATE          57600
-#if defined(RADIO_TX16S)
+#if defined(RADIO_TX16S) || defined(RADIO_F16)
```

Comment:
@3djc Do you think this could apply for all colorlcd radios, or was there something specific to the TX16S here?

Reply:
It was initially specific to TX16S since it required (direct) serial port. Not too sure anymore it was a good idea anyway, since FrSky inner telem is 57k anyway, you don't gain anything to output it faster, but you might loose compatibility with stuff deisgned to handle native 57k frsky telem. It was likely only make it faster because we can (and I was likely the one guilty of that)

Reply:
Thanks for that JC... Will leave this as it is for now then for consistency with TX16S, and can be corrected in `main` for both afterwards if it really should be yanked out.